ko/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: news_post
+title: "루비 2.3.0-프리뷰1 릴리스"
+author: "naruse"
+translator: "wagurano"
+date: 2015-11-11 14:00:00 +0000
+lang: ko
+---
+
+루비 2.3.0-프리뷰1 출시를 발표하여 기쁩니다.
+
+루비 2.3.0-프리뷰1는 루비 2.3.0의 첫번째 프리뷰입니다.
+새로운 기능과 개선사항을 포함합니다.
+
+[프로즌 스트링 리터럴
+프라그마](https://bugs.ruby-lang.org/issues/11473)를 도입합니다.
+루비 2.1에서, `"str".freeze` 를 최적화하여 객체 할당을 줄였습니다.
+루비 2.3은 매직 코멘트와 커맨드 라인 옵션을 새로 도입하여
+소스 파일의 모든 스트링 리터럴을 프리즈(freeze)를 합니다.
+그리고 디버깅할 때, `--enable-frozen-string-literal-debug` 옵션을 사용하면
+`"can't modify frozen String"` (프로즌 스트링을 수정할 수 없음) 에러가 발생하는
+객체를 생성한 위치를 알려줍니다.
+
+[세이프 내비게이션 오퍼레이터](https://bugs.ruby-lang.org/issues/11537)는
```

Comment:
At least, '오퍼레이터' can be written as '연산자'. I didn't think up appropriate words for '세이프 내비게이션', but someone uses '안전운항' for 'safe navigation', it may be a hint.